src/festim/initial_condition.py

```diff
@@ -98,3 +106,35 @@ def create_expr_fenics(self, mesh, function_space):
                 self.value(**kwargs),
                 get_interpolation_points(function_space.element),
             )
+
+
+def read_function_from_file(
+    filename: str, name: str, timestamp: int | float, family="P", order: int = 1
+) -> fem.Function:
+    """
+    Read a function from a file
+
+    note::
+        The function is read from a file using adios4dolfinx. For more information
+        see the [adios4dolfinx documentation](https://jsdokken.com/adios4dolfinx/README.html).
```

Comment:
could add a note here that the mesh in the file must be the same as the simulation where it will be used?